app/tests/spec/views/form.js

```diff
@@ -410,87 +410,64 @@ define(function (require, exports, module) {
       });
     });
 
-    describe('validateEmail', function () {
-      it('returns false if an empty email', function () {
+    describe('validate - mail', function () {
+      it('not valid if an empty email', function () {
         view.$('#email').val('');
-        assert.isFalse(view.validateEmail('#email'));
-        assert.isFalse(view.isElementValid('#email'));
+        assert.isTrue(AuthErrors.is(
+          view.$('#email').validate(), 'EMAIL_REQUIRED'));
       });
 
-      it('returns false if an invalid email', function () {
+      it('not valid if an invalid email', function () {
         view.$('#email').val('invalid');
-        assert.isFalse(view.validateEmail('#email'));
-        assert.isFalse(view.isElementValid('#email'));
+        assert.isTrue(AuthErrors.is(
+          view.$('#email').validate(), 'INVALID_EMAIL'));
       });
 
-      it('returns true if a valid email', function () {
+      it('valid if a valid email', function () {
         view.$('#email').val('testuser@testuser.com');
-        assert.isTrue(view.validateEmail('#email'));
-        assert.isTrue(view.isElementValid('#email'));
+        assert.notOk(view.$('#email').validate());
```

Comment:
Just to follow up on my earlier comment about this method, it makes these tests read quite unintuitively for instance.